### DISCUSSION TITLE
Get ride of LocalResource-Warning

### DISCUSSION THREAD

**Discussion by simon-bachhuber** [fonsp/Pluto.jl#2074] [Questions about using Pluto] [answered]
Hey guys,

how can i disable the LocalResource-Warning that wants me to save my files to imgur. It does not seem to disappear and it is very ugly.

Thanks :)

**Comment by holomorphism** [comment, marked as answer]
Hi.

I thought this problem was solved in PlutoUI v0.7.35.
https://github.com/JuliaPluto/PlutoUI.jl/issues/197#issuecomment-1048927855

I just checked and the warning message did not appear in my environment.
I think the version of PlutoUI in your notebook may be out of date.
You might try updating the package from the check mark to the right of `using PlutoUI`.
![LocalResource](https://user-images.githubusercontent.com/3076585/166085464-ca05ed38-8668-4a6b-897e-bb6366a50ed3.png)
